From c0f6e1ee2b931622ad9940b79556917e46309db3 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Sun, 2 Mar 2014 20:32:43 -0800
Subject: ASoC: simple-card: tidyup cpu/codec dai_fmt settings for non-DT

30d0341e7da0c012f64fb290dd1153360fb49a8d
(ASoC: simple-card: simplify the daifmt code)
simplify cpu/codec dai_fmt which consists from
dai specific format + common format.
But, it didn't care about non-DT case.
This patch fixes it

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 81985bd63f6f1711035aa8d97caf2fb203250a95)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 sound/soc/generic/simple-card.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sound/soc/generic/simple-card.c b/sound/soc/generic/simple-card.c
index 034a2b73f6c1..7cabcc5c8703 100644
--- a/sound/soc/generic/simple-card.c
+++ b/sound/soc/generic/simple-card.c
@@ -307,6 +307,9 @@ static int asoc_simple_card_probe(struct platform_device *pdev)
 						sizeof(priv->cpu_dai));
 		memcpy(&priv->codec_dai, &cinfo->codec_dai,
 						sizeof(priv->codec_dai));
+
+		priv->cpu_dai.fmt	|= cinfo->daifmt;
+		priv->codec_dai.fmt	|= cinfo->daifmt;
 	}
 
 	/*
-- 
2.1.2

